stratum/CMakeLists.txt

```diff
@@ -350,58 +350,94 @@ target_link_libraries(stratum PUBLIC
     p4runtime_proto
 )
 
+target_link_libraries(stratum PUBLIC pthread)
+
+# tdi
 find_library(LIBTDI tdi ${SDE_INSTALL_DIR}/lib)
 if(NOT LIBTDI)
     message(FATAL_ERROR "Cannot find library: tdi")
 endif()
 
+add_library(tdi SHARED IMPORTED)
```

Comment:
tdi and tdi_json_parser are no longer dependencies for Tofino. Only driver and target_sys

Reply:
I'll have to get a newer Tofino SDE before I can do this.